api/backups.go

```diff
@@ -0,0 +1,69 @@
+package api
+
+import (
+	"archive/zip"
+	"bytes"
+	"context"
+	"encoding/json"
+	"path/filepath"
+	"time"
+
+	"github.com/chadweimer/gomp/fileaccess"
+)
+
+func (h apiHandler) CreateBackup(_ context.Context, _ CreateBackupRequestObject) (CreateBackupResponseObject, error) {
+	// Export recipes
+	exportedRecipes, err := h.db.Backups().ExportRecipes()
```

Comment:
What about the idea of just always using sqlite as the export format? Read all tables from current DB and insert into a new sqlite DB?